tests/test_tiling_segmentation.py

```diff
@@ -260,8 +260,8 @@ def test_tiling_inference(self):
         cfg = DictConfig(cfg)
         tiling(cfg)
         out_tiles = [
-            (Path("data/tiles/test_inference/tiles32_RGBbands/trn/23322E759967N_clipped_1m_1of2/images"), 99),
-            (Path("data/tiles/test_inference/tiles32_RGBbands/inference/23322E759967N_clipped_1m_2of2/images"), 176),
+            (Path("data/tiles/test_inference/tiles32_123bands/trn/23322E759967N_clipped_1m_1of2/images"), 99),
+            (Path("data/tiles/test_inference/tiles32_123bands/inference/23322E759967N_clipped_1m_2of2/images"), 176),
```

Comment:
A little bit confusing for a notation if I was new I would not know what mean `123bands`

Reply:
Since GDL cannot know which spectral bands are present in multibands files, I see no other way around...